pylearn2/devtools/nan_guard.py

```diff
@@ -1,143 +0,0 @@
-"""
```

Comment:
Keep that file. Make it print a deprecation warning at import. Make it import the Theano version of nanguardmode so that old code continue to work for some time.